app/src/main/java/com/github/steroidteam/todolist/model/todo/TodoList.java

```diff
@@ -112,6 +112,15 @@ public TodoList sortByDate() {
         return this;
     }
 
+    public TodoList sortByDone() {
+        Collections.sort(
+                list,
+                (task, t1) -> {
+                    return Boolean.compare(task.isDone(), t1.isDone());
```

Comment:
I would like to know if this will change the order of the tasks in which they were added (I mean between non-done tasks, and otherwise between done tasks)  inside of a todolist and if yes, if it is acceptable or not.

Reply:
I did like Noah, so yes it changes the order of the task. But I think it is well handled in the ParentAdapter